ship.config.js

```diff
@@ -1,6 +1,5 @@
 const fs = require('fs');
 const path = require('path');
-// const getLatestVersion = require('latest-version');
```

Comment:
when updating with learn, the version numbers of all examples will be updated, but are the dependencies also updated automatically? where's the code for that, or is it just because it's in the workspace they get updated too?

Reply:
This is using the same strategy as [Autocomplete](https://github.com/algolia/autocomplete) which does update all the libraries deps in the examples with Lerna.